schemas/job_template_locale.yaml

```diff
@@ -5,10 +5,12 @@ job_template_locale:
   properties:
     id:
       type: string
+      nullable: true
```

Comment:
hm is this also nullable, this is not a locale id, but the id of the "relationship" resource?